pom.xml

```diff
@@ -58,7 +58,7 @@
     <!-- <maven.compiler.useIncrementalCompilation>false</maven.compiler.useIncrementalCompilation> -->
 
     <!-- thirdparty projects -->
-    <javaVersion>1.8</javaVersion>
+    <javaVersion>11</javaVersion>
```

Comment:
Does this mean we need to upgrade to jdk11(including both ci build env and local dev env)?

Reply:
It's only your local dev env that will need to be updated.
Whereas the ci build env is expected to continue working without any changes necessary. The image contains both OpenJDK8 and OpenJDK11.